### PR TITLE
Cb 10 call stock api

### PR DESCRIPTION
**Pull request by vishnubapana**
* **Please check if the PR fulfills these requirements**
- [ ] The pull request title contains the ticket number in the title. For example [CB-1]

* **What kind of change does this PR introduce?** (Bug fix, new feature...)
This PR introduces a new feature call stock info api that return the data of a given stock name.

* **What is the current behavior? What does the code do before you made the new changes?** 
Code did not have this feature before. 

* **What is the new behavior?**
With this new addition stock data is returned by providing the name of the stock in the endpoint.

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR? Run a new command?)
No changes required.

* **Other information**:

### CODE REVIEW

File: backend/index.js [hunk 1]
```diff
@@ -10,6 +10,20 @@ app.get('/',(req,res)=>{
     res.send('Hello world');
 })
 
+app.get('/stockname/:NAME',(req, res)=>{
+    const headers = {headers:{'Authorization':process.env.QUOTE_API_KEY}}
+    let NAME = req.params.NAME
+    axios.get(`https://finnhub.io/api/v1/quote?symbol=${NAME}&token=bv6jgev48v6s9eue2ndg`, headers)
+    .then(function(response){
+        console.log(response);
+        res.status(200).json(response.date);
```
Comment (Codubee) [inline backend/index.js]: I ran this and didnt get any data back, it might be data instead of date here.

File: backend/index.js [hunk 2]
```diff
@@ -10,6 +10,20 @@ app.get('/',(req,res)=>{
     res.send('Hello world');
 })
 
+app.get('/stockname/:NAME',(req, res)=>{
+    const headers = {headers:{'Authorization':process.env.QUOTE_API_KEY}}
+    let NAME = req.params.NAME
```
Comment (Codubee) [inline backend/index.js]: One issue that I didnt know existed until I put the company name is that if the company name is lowercase, the API will not bring back any data. One thing you can do to insure that the name of the company is always upper case is to use the javascript [toUpperCase](https://www.w3schools.com/jsref/jsref_touppercase.asp) method

**Review by Codubee — CHANGES_REQUESTED** [review]
Great work Venkata! Left a few comments

File: backend/index.js (hunk 1 above)
Comment (vishnubapana) [reply backend/index.js]: Fixed the issue. Originally was just looking at console for output data, now I changed date to data and now the information is being presented on the webpage.

File: backend/index.js (hunk 2 above)
Comment (vishnubapana) [reply backend/index.js]: Added the method and both lowercase and uppercase will display output.

**Review by Codubee — APPROVED** [review]
LGTM

**Pull request merged**